demo/deno.json

```
@@ -9,7 +9,8 @@
     "preact/": "https://esm.sh/preact@10.15.1/",
     "preact-render-to-string": "https://esm.sh/*preact-render-to-string@6.2.0",
     "@preact/signals": "https://esm.sh/*@preact/signals@1.1.3",
-    "@preact/signals-core": "https://esm.sh/@preact/signals-core@1.2.3"
+    "@preact/signals-core": "https://esm.sh/@preact/signals-core@1.2.3",
+    "~/": "./"
```

```suggestion
    "@/": "./"
```
Fresh projects typically use `@/` instead of `~/` (see [Deno SaaSKit](https://github.com/denoland/saaskit/blob/1061b81d8fed9e23f7039bf2446629623fc4ac5f/deno.json#L20)). Let's stick with that convention. I also think it'd be a good idea to move this to the top of the `imports` object.